src/components/Components/FieldsList.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const FieldsList = ({ children }) => (
```

Comment:
Maybe `Fieldset` could be a more accurate name ?

I personnaly prefer [_ItemList_ over _ItemsList_](https://stackoverflow.com/questions/9784720/items-list-or-item-list), but it's a matter of personal preferences. Maybe we could defined a styleguide for naming our list.